ksy_schema.json

```diff
@@ -276,6 +276,7 @@
       "properties": {
         "id": {
           "type": "string",
+          "pattern": "^[a-z][a-z0-9_]*$",
           "description": "contains a string that matches /^[a-z][a-z0-9_]*$/ used to identify one attribute among others"
```

Comment:
Now that the pattern is included in the schema, I think it can be removed from the description text.

Reply:
Done!